### PR TITLE
Feature/ccsd 1277 change copywrite

### PR DESCRIPTION
**Pull request by JacobBriggsAckama**
change copywrite notice on the footer of the web page and print web page

### CODE REVIEW

**Comment by DSRU** [comment]
Please also update the hyperlink to "https://creativecommons.org/licenses/by-nc-sa/4.0/"

**Comment by JacobBriggsAckama** [comment]
> Please also update the hyperlink to "https://creativecommons.org/licenses/by-nc-sa/4.0/"

Sure, that makes sense.

**Review by DSRU — APPROVED** [review]

File: app/views/shared/_print_footer.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <footer class="<%= (last ? 'sticky_footer footer' : 'footer--print footer') %>">
   <div class="copyright">
     <% if @footer && @footer.first_part %>
-      <%= image_tag "https://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png", class: 
+      <%= image_tag "https://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png", class:
```
Comment (joshmcarthur) [inline app/views/shared/_print_footer.html.erb]: I think we're going to want https://licensebuttons.net/l/by-nc-sa/4.0/88x31.png here to display the correct license badge for CC-by-SA 4.0.

Comment (joshmcarthur) [reply app/views/shared/_print_footer.html.erb]: Also, do we need a line break between `class:` and the class name?

**Review by joshmcarthur — CHANGES_REQUESTED** [review]

**Review by eoinkelly — APPROVED** [review]

File: app/views/shared/_print_footer.html.erb (hunk 1 above)
Comment (eoinkelly) [reply app/views/shared/_print_footer.html.erb]: fixed.

**Review by eoinkelly — APPROVED** [review]

**Pull request merged**